pages/_app.tsx

```diff
@@ -24,18 +24,21 @@ export default class MyApp extends App {
   render() {
     const { Component, pageProps } = this.props;
     return (
-      <Provider store={store}>
-        <MaterialUIThemeProvider theme={theme}>
-          <StyledComponentsThemeProvider theme={themeObject}>
-            <>
-              <CssBaseline />
-              <PageLayout>
-                <Component {...pageProps} />
-              </PageLayout>
-            </>
-          </StyledComponentsThemeProvider>
-        </MaterialUIThemeProvider>
-      </Provider>
+      <>
+        <title>Chapter</title>
```

Comment:
Why here instead of in the `<Head>` component?

Reply:
Good question, I'll move it to a Head component in here

Reply:
It looks like it was already moved from the `_document.tsx` file which was in a `<Head>` component already O_o

Reply:
Yeah, I moved it from there because of a Next.js error saying we could have a conflict and it would be unpredictable if we used `<Head><title></title></Head>` somewhere else
https://github.com/zeit/next.js/blob/master/errors/no-document-title.md

Reply:
it seems like `<Head><title></title></Head>` is the pattern they want us to use, just not in `_document.tsx`, right?

Reply:
Yeah, I forgot `<Head/>` here 🙈